conans/client/tools/system_pm.py

```diff
@@ -255,6 +309,9 @@ def installed(self, package_name):
                                  'findstr /c:"1 packages installed."' % package_name, None)
         return exit_code == 0
 
+    def _parse_package(self, package, arch):
+        return package
```

Comment:
Can we move this default implementation of `_parse_package` to the parent class (`BaseTool`). It is repeated several times.